shell-completion/bash/journalctl

```diff
@@ -110,6 +110,9 @@ _journalctl() {
             --identifier|-t)
                 comps=$(journalctl -F 'SYSLOG_IDENTIFIER' 2>/dev/null)
                 ;;
+            --exclude-identifier|-T)
+                comps=$(journalctl -F 'SYSLOG_IDENTIFIER' 2>/dev/null)
```

Comment:
Not a matter, but can be merged with the above case `--identifier|-t`.

Reply:
Anyways, merged. Thanks